packages/components/src/box-control/stories/index.story.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import BoxControl from '../';
+import type { BoxControlValue } from '../types';
+
+const meta: Meta< typeof BoxControl > = {
+	title: 'Components (Experimental)/BoxControl',
+	component: BoxControl,
+	argTypes: {
+		onChange: { action: 'onChange' },
```

Comment:
Instead of having specific configuration for the `onChange` prop, a more generic way to achieve the same result is to tell Storybook that all props starting with `on` should be treated as actions — you refer to other Storybook files ([example](https://github.com/WordPress/gutenberg/blob/f67c38677442518e65e338849eaa3fc435773a5c/packages/components/src/angle-picker-control/stories/index.story.tsx#L24)) to see how we usually add this configuration

Reply:
Good to know, thanks!